examples/minimal.rs

```diff
@@ -118,8 +124,10 @@ async fn start_single_node(
     contacts: Vec<SocketAddr>,
     ip: Option<IpAddr>,
     port: Option<u16>,
-) {
-    let _contact = start_node(0, first, contacts.into_iter().collect(), ip, port).await;
+) -> JoinHandle<()> {
+    let (_contact, handler): (SocketAddr, JoinHandle<()>) =
```

Comment:
Again, rust can infer the type, no need to specify them. Also `handler` -> `handle`.